PC/layout/support/pymanager.py

```diff
@@ -0,0 +1,249 @@
+            if X_TAG:
+                STD_ALIAS.extend([
+                    {**base, "name": f"{prefix}{X_TAG}.exe"},
+                    {**base, "name": f"{prefix}{X_ARCH_TAG}.exe"},
+                ])
+
+    STD_PEP514.append({
+        "kind": "pep514",
+        "Key": rf"{COMPANY}\{ID_TAG}",
+        "DisplayName": f"{DISPLAY_NAME} {DISPLAY_VERSION}",
+        "SupportUrl": "https://www.python.org/",
+        "SysArchitecture": SYS_ARCH,
+        "SysVersion": VER_DOT,
+        "Version": FULL_VERSION,
+        "InstallPath": {
+            "_": "%PREFIX%",
+            "ExecutablePath": f"%PREFIX%{TARGET}",
+            # WindowedExecutablePath is added below
+        },
+        "Help": {
+            "Online Python Documentation": {
+                "_": f"https://docs.python.org/{VER_DOT}/"
+            },
+        },
+    })
+
+    STD_START.append({
+        "kind": "start",
+        "Name": f"{DISPLAY_NAME} {VER_DOT}{DISPLAY_SUFFIX}",
+        "Items": [
+            {
+                "Name": f"{DISPLAY_NAME} {VER_DOT}{DISPLAY_SUFFIX}",
+                "Target": f"%PREFIX%{TARGET}",
+                "Icon": f"%PREFIX%{TARGET}",
+            },
+            {
+                "Name": f"{DISPLAY_NAME} {VER_DOT} Online Documentation",
+                "Icon": r"%SystemRoot%\System32\SHELL32.dll",
+                "IconIndex": 13,
+                "Target": f"https://docs.python.org/{VER_DOT}/",
+            },
+            # IDLE and local documentation items are added below
+        ],
+    })
+
+    if TARGETW:
+        STD_PEP514[0]["InstallPath"]["WindowedExecutablePath"] = f"%PREFIX%{TARGETW}"
+
+    if ns.include_idle:
+        STD_START[0]["Items"].append({
+            "Name": f"IDLE {VER_DOT}{DISPLAY_SUFFIX}",
+            "Target": f"%PREFIX%{TARGETW or TARGET}",
+            "Arguments": r'"%PREFIX%Lib\idlelib\idle.pyw"',
+            "Icon": r"%PREFIX%Lib\idlelib\Icons\idle.ico",
+            "IconIndex": 0,
+        })
+        STD_START[0]["Items"].append({
+            "Name": f"PyDoc {VER_DOT}{DISPLAY_SUFFIX}",
+            "Target": f"%PREFIX%{TARGET}",
+            "Arguments": "-m pydoc -b",
+            "Icon": r"%PREFIX%Lib\idlelib\Icons\idle.ico",
+            "IconIndex": 0,
+        })
+
+    if ns.include_html_doc:
+        STD_PEP514[0]["Help"]["Main Python Documentation"] = {
+            "_": rf"%PREFIX%Doc\html\index.html",
+        }
+        STD_START[0]["Items"].append({
+            "Name": f"{DISPLAY_NAME} {VER_DOT} Manuals{DISPLAY_SUFFIX}",
+            "Target": r"%PREFIX%Doc\html\index.html",
+        })
+    elif ns.include_chm:
+        STD_PEP514[0]["Help"]["Main Python Documentation"] = {
+            "_": rf"%PREFIX%Doc\{PYTHON_CHM_NAME}",
+        }
+        STD_START[0]["Items"].append({
+            "Name": f"{DISPLAY_NAME} {VER_DOT} Manuals{DISPLAY_SUFFIX}",
+            "Target": "%WINDIR%hhc.exe",
+            "Arguments": rf"%PREFIX%Doc\{PYTHON_CHM_NAME}",
+        })
```

Comment:
I had thought we removed CHM ages ago?

Reply:
This script can be used to repackage releases a long way back, so it still has the support in there. Even though it lives in the repo alongside a particular version, it is pretty independent of the actual version it's laying out.